actions/docker/metadata-action/v4/action-types.yml

```
@@ -0,0 +1,34 @@
+# See https://github.com/krzema12/github-actions-typing
+inputs:
+  images:
+    type: list
+    separator: '\n'
+    list-item:
+      type: string
+  tags:
+    type: list
+    separator: '\n'
+    list-item:
+      type: string
+    required: false
```

In the typings file we just specify the types, and the input being required or not is inferred automatically from actions.yml. We should perhaps fail on unknown attributes when deserializing this YAML here

---

Created https://github.com/krzema12/github-workflows-kt/issues/719